modules/install-and-configure-oadp-kubevirt.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="install-and-configure-oadp-kubevirt_{context}"]
+= Installing and configuring {oadp-short} with {VirtProductName}
+
+As a cluster administrator, you install {oadp-short} by installing the {oadp-short} Operator.
+
+The Operator installs link:https://velero.io/docs/v{velero-version}[Velero {velero-version}].
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Install the {oadp-short} Operator according to the instructions for your storage provider.
```

Comment:
This procedure doesn't make sense to me. 

Description: "As a cluster administrator, you install {oadp-short} by installing the {oadp-short} Operator."

Step 1: "Install the {oadp-short} Operator according to the instructions for your storage provider."

It doesn't actually tell me how to do what the title indicates.

Reply:
It has been approved multiple times